src/pages/Explorer/Components/Actions.svelte

```diff
@@ -112,8 +117,8 @@
       {/if}
       <Svg id="rocket" w="16" class="btn $style.svg" on:click={onVote} />
       <Svg id="share-dots" w="16" class="btn $style.svg" on:click={onShare} />
-      {#if isModerator}
-        <Svg id="eye-crossed" w="16" class="btn $style.svg" />
+      {#if $currentUser.isModerator && !isOwner}
```

Comment:
1. `$currentUser` might be `null` for the anonymous user;
2. `!isOwner` is already implied since the code is located in the `:else` branch of `{#if isOwner}` statement